UnitTests/GitCommands.Tests/RevisionReaderTests.cs

```diff
@@ -115,8 +117,11 @@ public void BuildArguments_check_parameters(RefFilterOptions refFilterOptions, s
         public void TryParseRevisionshould_return_false_if_argument_is_invalid()
         {
             ArraySegment<byte> chunk = null;
+            RevisionReader reader = new(new GitModule(""), _logOutputEncoding, _sixMonths);
```

Comment:
I prefer this over `new(new(""), ...)`. No need to change for testcases.

Reply:
I like the compactness and this is tests. But in previous reviews we have often pointed to including the type, so I changed